dev-python/nilearn/nilearn-0.1.4.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
AFAIK it is not, at least I know nothing about it. https://www.gentoo.org/support/news-items/2015-07-25-python-targets.html only tells that 3.4 is gentoos new Python 3 default.